### ISSUE TITLE
Can not get width and height of 1600 and/or 2048 thumbnail sizes.

### ISSUE THREAD

**Issue by fvsnippets** [boncey/Flickr4Java#178]
Would be good to have available the height and width for all sizes (including 1600 and 2048).
i.e. (com.flickr4java.flickr.photos.PhotoUtils):

```
   urlTmp = photoElement.getAttribute("url_h");
   if (urlTmp != null && urlTmp.startsWith("http")) {
       Size sizeT = new Size();
       sizeT.setLabel(Size.THUMB);
       sizeT.setSource(urlTmp);

       sizeT.setWidth(photoElement.getAttribute("width_h"));
       sizeT.setHeight(photoElement.getAttribute("height_h"));

       sizes.add(sizeT);
   }
```

And the same goes for url_k (replacing the "_letter" suffix).

Thank you.

**Comment by fvsnippets** [comment]
I've just seen that [flickr search](https://www.flickr.com/services/api/flickr.photos.search.html) doesn't mention url_h or url_k. They are not documented, but they are working.

**Comment by boncey** [comment]
Just noticed that this was fixed ages ago in #283 .
